src/App.js

```diff
@@ -1,13 +1,146 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoSearch from './components/TodoComponents/TodoSearch';
+
+let todoData = {}
+const myLastStorage  = localStorage.getItem("lsTodoData");
+
+if (!myLastStorage){
+  todoData = {todos: [
+    {
+      task: 'Organize Crime',
+      id: 1528817077286,
+      completed: false,
+      display: true
+    },
+    {
+      task: 'Bake Cookies',
+      id: 1528817084358,
+      completed: true,
+      display: true
+    },
+    {
+      task: 'Eat salad',
+      id: 1528817084558,
+      completed: false,
+      display: true
+    },
+    {
+      task: 'Throw up from stress (a little)',
+      id: 1528817484358,
+      completed: true,
+      display: true
+    },
+    {
+      task: 'Fluff Pillows',
+      id: 1528817224358,
+      completed: false,
+      display: true
+    }
+  ], 
+  inputText: '',
+  searchText: '',
+  };
+
+  localStorage.setItem("lsTodoData", JSON.stringify(todoData));
```

Comment:
Check out componentdidmount for stuff like this, we'll be discussing it next week in depth